target/linux/ramips/dts/mt7621_cudy-wr2100.dts

```diff
@@ -0,0 +1,241 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "cudy,wr2100", "mediatek,mt7621-soc";
+	model = "Cudy WR2100";
+
+	aliases {
+		led-boot = &led_blue;
+		led-failsafe = &led_blue;
+		led-running = &led_blue;
+		led-upgrade = &led_blue;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_blue: blue {
+			label = "green:blue";
```

Comment:
Honestly?

Reply:
i shit you not: these are the labels from the stock device tree based on MTK. i only removed the `oem:` at the beginning.
so far, there's only been blue and red - so i assume, the green is garbage.
once there's something bootable, i'll confirm the colours.

Reply:
```
echo 255 > oem\:green\:red/brightness
echo 255 > oem\:green\:blue/brightness
```
they turn on a red and a blue LED. the only greens are the ethernet ports.
are `red` and `blue` sufficing labels?

Reply:
well, just because the stock device tree is nonsense, I don't want to transfer that into our project. Just properly name these with color:function due to what they are supposed to do

Reply:
at no point did i say we should do that.

i will ask cudy about how they intended the LEDs to be used.
as of today, i don't know whether the red/blue LEDs have a specific purpose.
atm, i can only say that, in the stock firmware, `sysupgrade` cycles through >all< LEDs (LAN1,2,3,4,red,blue,WAN), the blue LED is active without a recognizable pattern, and the red LED hardly ever goes on.

Reply:
let's get the wifi working first.
without wifi, there's hardly any use talking about the intended purpose of LEDs.

Reply:
@adschm hey :-)
i played around with the stock firmware and, afaics, the blue+red LEDs are used in combination to reflect the detected uplink status.

red => no internet, blue -> internet

Reply:
i will name the LEDs `uplink`. is there a more 'openwrt' way of doing this? do you prefer another name?

Reply:
Well, I'd have thought of red:internet and blue:internet, but it's essentially a matter of taste.

Reply:
i've gone with your suggestion. resolve this conversation?